dev/components/installPluginControls.js

```diff
@@ -6,6 +6,40 @@ class InstallPluginControls extends React.Component {
         this.state = {
             isOpen: this.props.isOpen
         };
+
+        FormItInterface.SubscribeMessage("FormIt.Message.kInstallPlugin", (data) => {
+            let jsonObj = JSON.parse(data);
+            if (this.state.installUrl.startsWith(jsonObj.payload)) {
+                this.setState({installUrl: ''});
+                if(this.notificationHandle)
+                    FormIt.UI.CloseNotification(this.notificationHandle);
+                FormIt.UI.ShowNotification(
+                    "Plugin Installed",
+                    FormIt.NotificationType.Success,
+                    3000
+                );
+            }
+        });
+    }
+
+    notificationHandle
```

Comment:
This looks like it's just floating, is it supposed to be here?

Reply:
its used to remove the notification if success happens before the bubble timeout finishes